sympy/stats/drv_types.py

```diff
@@ -1,9 +1,10 @@
 from __future__ import print_function, division
 
 from sympy.stats.drv import SingleDiscreteDistribution, SingleDiscretePSpace
-from sympy import factorial, exp, S, sympify
+from sympy import factorial, exp, S, sympify, floor, sqrt, log, uppergamma, sign
 from sympy.stats.rv import _value_check
 from sympy.sets.sets import Interval
+from mpmath import pi
```

Comment:
I advise against importing a name that shadows SymPy's name. What if someone needs SymPy's pi in this module? 
SymPy's pi can be used in numerical context too. So, adding pi to the imports "from sympy" on line 4 would be enough.